Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -223,24 +228,27 @@ class FakespotViewController:
             betaLabel.topAnchor.constraint(equalTo: betaView.topAnchor, constant: UX.betaVerticalSpace),
             betaLabel.leadingAnchor.constraint(equalTo: betaView.leadingAnchor, constant: UX.betaHorizontalSpace),
             betaLabel.trailingAnchor.constraint(equalTo: betaView.trailingAnchor, constant: -UX.betaHorizontalSpace),
-            betaLabel.bottomAnchor.constraint(equalTo: betaView.bottomAnchor, constant: -UX.betaVerticalSpace)
+            betaLabel.bottomAnchor.constraint(equalTo: betaView.bottomAnchor, constant: -UX.betaVerticalSpace),
+            betaLabel.widthAnchor.constraint(greaterThanOrEqualToConstant: UX.betaMinWidth)
         ])
-
-        adjustLayout()
     }
 
     private func adjustLayout() {
         let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
+        let titleTextWidth = widthOfString(titleLabel.text!, usingFont: titleLabel.font)
+        let maxLabelWidth = headerView.frame.width - widthOfString(betaLabel.text!, usingFont: betaLabel.font)
```

Comment:
The max label size is even smaller. You don't account for the space between the label and beta label nor for the border and padding around the beta label.